NanoProd/crab/Run3_2023/data.yaml

```diff
@@ -0,0 +1,41 @@
+config:
+  params:
+    sampleType: data
+    era: Run3_2023
+
+EGamma0_v1_Run2022C: /EGamma0/Run2023C-22Sep2023_v1-v1/MINIAOD
```

Comment:
I prefer to keep order in our names as close as possible to the original names on DAS. E.g. `EGamma0_Run2022C_v1` etc.